dev/VSIX/ItemTemplates/Neutral/CSharp/TemplatedControl/CustomControl.cs

```diff
@@ -12,13 +12,12 @@
 // To learn more about WinUI, the WinUI project structure,
 // and more about our project templates, see: http://aka.ms/winui-project-info.
 
-namespace $rootnamespace$
+namespace $rootnamespace$;
+
+public sealed class $safeitemname$ : Control
```

Comment:
```suggestion
public sealed partial class $safeitemname$ : Control
```
We should mark all classes in the templates `partial` now for AOT (and other source generators)

Reply:
`partial` added